sdk/script/account-manager.ts

```diff
@@ -9,12 +8,12 @@ var packageJson = require("../../package.json");
 
 declare var fs: any;
 
-if (typeof window === "undefined") {
-    fs = require("fs");
+if (typeof window === `undefined`) {
```

Comment:
I'm just wondering whether we should from now on use template strings for everything, or only when it is required.